src/constants/java/net/fabricmc/yarn/constants/MiningLevels.java

```diff
@@ -0,0 +1,37 @@
+package net.fabricmc.yarn.constants;
+
+public final class MiningLevels {
```

Comment:
Imo we need a class-level doc, like one linking to `net.minecraft.item.ToolMaterial#getMiningLevel()`